.github/workflows/ci.yaml

```diff
@@ -58,8 +58,8 @@ jobs:
 
     - name: Build
       run: |
-        nix-shell --arg withoutDevTools true --run 'cabal update'
-        nix-shell --arg withoutDevTools true --run 'cabal build ${{ matrix.package }}'
+        nix-shell --pure --arg withoutDevTools true --run 'cabal update'
+        nix-shell --pure --arg withoutDevTools true --run 'cabal build ${{ matrix.package }}'
```

Comment:
What's the meaning of `pure` here?

Reply:
It does not use anything from the host environment. I think hai added it to not use git from mac or so..at least he added git, which was seemingly the only thing used from our hosts.